client/js/_author/components/assessments/question_types/question_common/single_feedback.jsx

```diff
@@ -26,7 +26,7 @@ export default class SingleFeedback extends React.Component {
     *  to qbank. We make it here instead of in the editor, because the editor
     *  needs uploadScopeId in the select function.
     */
-    this.editorScopeId = _.get(this.props.feedback, 'answerId', guid());
```

Comment:
Alright. I don't think that a null answer id should ever get to the components, because of stuff like this where we actually want it to behave like it's undefined, but if that's how we're doing feedback for new questions then ok.